Editor/Analytics/AmplitudeEditorLogger.cs

```diff
@@ -59,190 +61,200 @@ public void LogOpenProject()
             });
         }
 
         public void LogClearLocalCache()
         {
-            LogEvent(Constants.EventName.CLEAR_LOCAL_CACHE);
+            LogEvent(EventName.CLEAR_LOCAL_CACHE);
         }
 
         public void LogViewPrivacyPolicy()
         {
-            LogEvent(Constants.EventName.PRIVACY_POLICY);
+            LogEvent(EventName.PRIVACY_POLICY);
         }
 
         public void LogShowInExplorer()
         {
-            LogEvent(Constants.EventName.SHOW_IN_EXPLORER);
+            LogEvent(EventName.SHOW_IN_EXPLORER);
         }
 
         public void LogFindOutMore(HelpSubject subject)
         {
-            LogEvent(Constants.EventName.FIND_OUT_MORE, new Dictionary<string, object>
+            LogEvent(EventName.FIND_OUT_MORE, new Dictionary<string, object>
             {
-                { Constants.Properties.CONTEXT, helpDataMap[subject] }
+                { Properties.CONTEXT, helpDataMap[subject] }
             });
         }
 
         public void LogOpenSetupGuide()
         {
-            LogEvent(Constants.EventName.OPEN_SETUP_GUIDE);
+            LogEvent(EventName.OPEN_SETUP_GUIDE);
         }
 
         public void LogOpenIntegrationGuide()
         {
-            LogEvent(Constants.EventName.OPEN_INTEGRATION_GUIDE);
+            LogEvent(EventName.OPEN_INTEGRATION_GUIDE);
         }
 
         public void LogLoadQuickStartScene()
         {
-            LogEvent(Constants.EventName.LOAD_QUICK_START_SCENE);
+            LogEvent(EventName.LOAD_QUICK_START_SCENE);
         }
 
         public void LogOpenAvatarDocumentation()
         {
-            LogEvent(Constants.EventName.OPEN_AVATAR_DOCUMENTATION);
+            LogEvent(EventName.OPEN_AVATAR_DOCUMENTATION);
         }
 
         public void LogOpenAnimationDocumentation()
         {
-            LogEvent(Constants.EventName.OPEN_ANIMATION_DOCUMENTATION);
+            LogEvent(EventName.OPEN_ANIMATION_DOCUMENTATION);
         }
 
         public void LogOpenAvatarCreatorDocumentation()
         {
-            LogEvent(Constants.EventName.OPEN_AVATAR_CREATOR_DOCUMENTATION);
+            LogEvent(EventName.OPEN_AVATAR_CREATOR_DOCUMENTATION);
         }
 
         public void LogOpenOptimizationDocumentation()
         {
-            LogEvent(Constants.EventName.OPEN_OPTIMIZATION_DOCUMENTATION);
+            LogEvent(EventName.OPEN_OPTIMIZATION_DOCUMENTATION);
         }
 
         public void LogAvatarCreatorSampleImported()
         {
-           LogEvent(Constants.EventName.AVATAR_CREATOR_SAMPLE_IMPORTED);
+            LogEvent(EventName.AVATAR_CREATOR_SAMPLE_IMPORTED);
+        }
+
+        public void LogPackageInstalled(PackageCoreInfo packageInfo, bool force = false)
+        {
+            LogEvent(EventName.INSTALL_PACKAGE, new Dictionary<string, object>
+            {
+                { "id", packageInfo.Id },
+                { "name", packageInfo.Name },
+                { "url", packageInfo.Url },
+            }, force: force);
```

Comment:
I am not sure whether we should do this according to GDPR. Until and unless someone have enabled analytics we shouldn't send an event.

Reply:
It's not a GDPR concern since it's not in anyway personal data, but still, I understand the desire to discuss it, so let's do so tomorrow in stand-up.

Reply:
We discussed in stand-up and found a way to achieve it without skipping the acceptance 🙏